addons/website/views/website_visitor_views.xml

```diff
@@ -79,7 +79,7 @@
         <field name="name">website.visitor.view.kanban</field>
         <field name="model">website.visitor</field>
         <field name="arch" type="xml">
-            <kanban class="o_wvisitor_kanban">
+            <kanban create="0" class="o_wvisitor_kanban">
```

Comment:
Views follow ACLs, so I don't think you have to specify that.

If you connect as "website designer", since you don't have the rights to create, the create button will not be there anyway.